src/loc/bg-bg.ts

```diff
@@ -2,6 +2,18 @@ declare var define: any;
 
 define([], () => {
   return {
+  MyTeamsLoadingMessage: "loading your teams",
+  MyTeamsMessageDontHaveTeams: "You don't have any teams",
+  MyTeamsMessageError: "Something went wrong while loading your teams, please try later or refresh browser",
+  MyTeamsNoTeamsMessage: "You don't have any teams",
+  MyTeamsTeamChannelPublicMessage: "Public Channels",
+  MyTeamsTeamChannelTypeMessage: "Private Channels",
+  TeamChannelPickerFontIconFavoriteText: "Favorite",
+  TeamChannelPickerFontIconPrivateChannelTitle: "Private Channel",
+  TeamChannelPickerSugestionHeaderText: " \"Suggested Team Channels\"",
```

Comment:
Double quote escaping, possibly caused by a VS code extension? Applies to other languages as well